modules/ROOT/pages/custom-policy-set-authentication-example.adoc

```diff
@@ -0,0 +1,198 @@
+
+You can rename this alias if required. To have access to the repository, ProTeams needs to add the corresponding server in their Maven settings file:
+
+[source,XML,linenums]
+----
+<servers>
+      <server>
+         <id>releases-ee</id>
+         <username>your_username</username>
+         <password>your_password</password>
+       </server>
+</servers>
+----
+
+[[add_dependencies_in_pom]]
+=== Add Dependencies in pom.xml File
+
+ProTeams then adds dependencies in the `pom.xml` file for the policy to perform specific actions, such as the Event Authentication and the HTTP Policy Transform extensions:
+
+[source,XML,linenums]
+----
+    <dependencies>
+        <dependency>
+            <groupId>com.mulesoft.anypoint</groupId>
+            <artifactId>mule-event-authentication-extension</artifactId>
+            <version>${authenticationExtensionVersion}</version>
+            <classifier>mule-plugin</classifier>
+        </dependency>
+        <dependency>
+            <groupId>com.mulesoft.anypoint</groupId>
+            <artifactId>mule-http-policy-transform-extension</artifactId>
+            <version>${httpPolicyTransformVersion}</version>
+            <classifier>mule-plugin</classifier>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.mule.connectors</groupId>
+                    <artifactId>mule-http-connector</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+    </dependencies>
+----
+
+=== Add Logic in template.xml File
+
+ProTeams now needs to add the functionality logic that the policy must perform in the `template.xml file`, such as the ID mapping for ProTeams.
+
+As ProTeams value mapping is not available in a Mule Extension, they must create and add their own custom module.
+
+[[upload_to_exchange]]
+== Uploading to Exchange
+
+To upload the custom policy to Exchange, ProTeams must verify that their `pom.xml` file is correctly configured for their organization.
+
+=== Task Prerequisites
+
+Before uploading their policy to Exchange, they must verify that:
+
+* A new server is created in their Maven `settings.xml` file, with a unique ID, and username and password of the Exchange credentials for their organization.
+* The URL matches their region. 
++
+Warning: The template has the URL that corresponds to the US region. If your account does not belong to this region, ensure that you change it.
+* The repository ID matches the one from the Maven settings. 
++
+*Tip:* You can define an alias to that repository using the name tag.
+
+[source,XML,linenums]
+----
+<servers>
+      <server>
+         <id>your_server_id</id>
+         <username>your_organization_username</username>
+         <password>your_organization_password</password>
+       </server>
+</servers>
+----
+
+Next, ProTeams goes to their policy `pom.xml` file` and locates the `distributionManagement` section:
```

Comment:
Is this still a prerequisite, or is this the actual start of uploading to Exchange? Clarify.

Reply:
Done! Thanks!